language-support/ts/daml-types/tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "declaration": true,
     "sourceMap": true
   },
-  "files": ["index.ts"]
+  "files": ["index.ts", "index.test.ts"]
```

Comment:
I think we can remove this line completely. And similarly in the other `tsconfig.json` files.

Reply:
I removed all of them now.